integration-tests/src/test/scala/tech/beshu/ror/integration/suites/base/support.scala

```diff
@@ -29,51 +33,84 @@ object support {
     extends RorConfigFileNameProvider
       with MultipleClientsSupport
       with TestSuiteWithClosedTaskAssertion
-      with ForAllTestContainer {
+      with ForAllTestContainer
+      with BeforeAndAfterAll {
     this: Suite with EsClusterProvider with ESVersionSupport =>
 
     override lazy val container: EsClusterContainer = clusterContainer
 
+    override protected def afterAll(): Unit = {
+      super.afterAll()
+      pruneDockerImages()
```

Comment:
Adding the new test with apt-based ES caused the jobs to fail - not enough space on runner. Probably caused by downloading ~700 MB of ES package. In the first step I added cleaning of the resulting Docker images in newly added tests, but it was not enough. So I added pruning unused Docker images after each test suite.

Reply:
1. Do we have ideas how to optimize it?
  - Do we have to use the Ubuntu image? Maybe we can use a thinner alternative
  - Are we sure we don't prune this image between the tests?
  - Any other ideas for current or/and future improvements in this matter?
2. The thing Simone mentioned is important. We want to release a new version as soon as we can, so IMO it's better to address this issue now. Let's compare the dates of releasing the latest versions (the docker images and apt packages). Maybe we could check if there is a way to build the custom ES images, and if it's not, the test could be skipped. WDYT?